src/matchmaking/tournament/roundrobin/roundrobin.cpp

```diff
@@ -118,7 +118,7 @@ void RoundRobin::updateSprtStatus(const std::vector<EngineConfiguration>& engine
     if (!sprt_.isValid()) return;
 
     const auto stats = result_.getStats(engine_configs[0].name, engine_configs[1].name);
-    const auto llr   = sprt_.getLLR(stats.wins, stats.draws, stats.losses);
+    const auto llr   = sprt_.getLLR(stats.penta_WW, stats.penta_WD, stats.penta_WL, stats.penta_DD, stats.penta_LD, stats.penta_LL);
```

Comment:
i think this doesnt work correctly, since it can be either cutechess or fastchess here..
something like
`const auto llr = tournament_options_.report_penta ? sprt_.getLLR(...) : sprt_.getLLR(...);`

Reply:
ahh youre right, im assuming tournament_option_.report_penta is set to 0 when cutechess output is enabled?

Reply:
@Disservin fixed :D